src/ant/src/AntennaChecker.cc

```diff
@@ -478,24 +478,24 @@ void AntennaChecker::calculatePAR(GateToLayerToNodeInfo& gate_info)
 // calculate CAR and CSR of wires and vias
 void AntennaChecker::calculateCAR(GateToLayerToNodeInfo& gate_info)
 {
-  for (auto& gate_it : gate_info) {
+  for (auto& [gate, layer_to_node_info] : gate_info) {
     NodeInfo sumWire, sumVia;
     // iterate from first_layer -> last layer, cumulate sum for wires and vias
     odb::dbTechLayer* iter_layer = min_layer_;
     while (iter_layer) {
-      if (gate_it.second.find(iter_layer) != gate_it.second.end()) {
+      if (layer_to_node_info.find(iter_layer) != layer_to_node_info.end()) {
         if (iter_layer->getRoutingLevel() == 0) {
-          sumVia += gate_it.second[iter_layer];
-          gate_it.second[iter_layer].CAR += sumVia.PAR;
-          gate_it.second[iter_layer].CSR += sumVia.PSR;
-          gate_it.second[iter_layer].diff_CAR += sumVia.diff_PAR;
-          gate_it.second[iter_layer].diff_CSR += sumVia.diff_PSR;
+          sumVia += layer_to_node_info[iter_layer];
```

Comment:
You should just store `layer_to_node_info[iter_layer]` in a variable between line 486 and 487 and then you can use it both the if and else. Plus probably save yourself some look ups.